src/components/sidebar/SideBar.js

```diff
@@ -0,0 +1,45 @@
+import { useState } from "react";
+import { FaHome, FaBars } from "react-icons/fa";
+import { ImBlogger2 } from "react-icons/im";
+import { NavLink } from "react-router-dom";
+import './SideBar.css'
+
+const SideBar = ({ children }) => {
+    const [isOpen, setIsOpen] = useState(false)
+    const toggle = () => setIsOpen(!isOpen)
+  const menuItems = [
+    {
+      path: "/",
+      name: "Blogs",
+      icon: <FaHome />,
+    },
+    {
+      path: "/create",
+      name: "CreateBlog",
+      icon: <ImBlogger2 />,
+    },
+  ];
```

Comment:
This can be outside of the Sidebar function

Reply:
ok. I will adjust this.